File: BackEnd/10_week_Algorithms/2.알고리즘의_연산량/2B_오름차순인가?/2B.py
# 오름차순인가?

n = int(input())
data = list(map(int, input().split()))

previous = 0
result = 'YES'

for i in range(0, n-1):
    if data[i+1] < data[i]:
        result = 'NO'
        break

print(result)

File: Algorithm/10-Week_Coding_Test/2.알고리즘의_연산량/2B_오름차순인가/2B.js
// 오름차순인가?

const fs = require("fs")
const data = fs.readFileSync("/dev/stdin").toString().split("\n")

const n = data[0]
const arr = data[1].split(" ").map(Number)

result = 'YES'

for (let i = 0; i < n-1; i++) {
  if (arr[i] > arr[i+1]) {
    result = 'NO'
    break
  }
}

console.log(result)
